docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -369,6 +369,40 @@ Teleport's smartcard certificates are only valid for a short period of time
 target Windows host disagree about the current time, the system may reject the
 authentication attempt.
 
+### Expired TLS certificate
```

Comment:
I wonder if we should find a way to combine this with the previous topic rather than treat it separately.

Reply:
> I wonder if we should find a way to combine this with the previous topic rather than treat it separately.

As a user of documentation, I would prefer to see those two issues categorized under the same subheading. Let me work up a proposal later on in the day.

Reply:
Happy to close this one if you prefer the work on #46534